pkg/asset/agent/manifests/infraenv.go

```diff
@@ -70,6 +71,11 @@ func (i *InfraEnv) Generate(dependencies asset.Parents) error {
 				},
 			},
 		}
+
+		// Use RpmArch for infraEnv.Spec.CpuArchiteture. Default is amd64 or "", needs to be x86_64 or aarch64
```

Comment:
It seems like we might be halfway between two different approaches here. I think we should pick either:

1. Historically (pre https://github.com/openshift/assisted-service/pull/4441), the accepted values in ZTP were `x86_64` or `arm64`, so we should generate the manifest with those values and convert to RpmArch in ignition.go; or
2. We should be consistent in generating rpm-arch strings here, and also convert from docker-arch strings if necessary in `Load()` so that we only ever have rpm-arch in this field, and no conversion is required in ignition.go.

In either case we should have validation in `finish()` to check that only valid values have been provided in the case where we are `Load()`ing a user-provided manifest from disk. That makes me lead toward the second one.